### PR TITLE
Update activate-lagged-db-copies.md

### PR DESCRIPTION
**Pull request by andreig2735**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/high-availability/manage-ha/activate-lagged-db-copies.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ Looking for other information related to lagged mailbox database copies? Check o
    ```powershell
    Eseutil.exe /r eXX /a
    ```
+Note: if the database being recovered is "out of place", make sure to specify the logfile,checkpoint and database paths in the eseutil command, for example:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/high-availability/manage-ha/activate-lagged-db-copies.md:68]:
```suggestion
**Note**: If the database being recovered is "out of place", make sure to specify the log file, checkpoint, and database paths in the eseutil command, for example:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**